services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -672,12 +677,18 @@ public static Builder builder() {
    * Create a new instance of this class by passing in a formatted valid JSON String.
    *
    * @param json a formatted valid JSON string defining a RouteOptions
+   * @param accessToken the Mapbox Access Token
    * @return a new instance of this class defined by the values passed inside this static factory
    *   method
    * @see #fromUrl(URL)
    */
   @NonNull
-  public static RouteOptions fromJson(String json) {
+  public static RouteOptions fromJson(@NonNull String json, @Nullable String accessToken) {
```

Comment:
just to clarify, is this method used anywhere except tests?
will we always have `accessToken=null` in the `DirectionsResponse `?

Reply:
This is a utility so that devs don't need to do `fromJson.toBuilder.accessToken.build` and it's clear that `accessToken` needs to be provided again, it's not serialized.